src/arrays.jl

```diff
@@ -74,7 +75,7 @@ function Base.convert(::Type{QA1}, A::QA2) where {QA1<:QuantityArray,QA2<:Quanti
     N = ndims(QA1)
 
     raw_array = Base.Fix1(convert, Q).(A)
-    output = QuantityArray(convert(constructor_of(V){Q,N}, raw_array))
+    output = QuantityArray(convert(constructorof(V){Q,N}, raw_array))
```

Comment:
Same here.

Reply:
What should be used instead?

Reply:
n.b., `array_type` is defined by:

https://github.com/SymbolicML/DynamicQuantities.jl/blob/151d3246071082be2edab606427e85e72a737a97/src/arrays.jl#L87

so it will throw an error if these type parameters are not available (like in a `UnionAll`)

Reply:
Similar for `quantity_type`:

https://github.com/SymbolicML/DynamicQuantities.jl/blob/151d3246071082be2edab606427e85e72a737a97/src/arrays.jl#L90

Reply:
I guess I could do something like:

```julia
    Q = quantity_type(QA1)
    D = dimension_type(QA1)
    V = array_type(QA1)

    output = QuantityArray(convert(V, ustrip(A)), convert(D, dimension(A)), Q)
```

and that would avoid the `constructorof`

Reply:
Fixed.